apps/stable_diffusion/src/pipelines/pipeline_shark_stable_diffusion_utils.py

```diff
@@ -203,7 +221,10 @@ def produce_img_latents(
         latent_history = [latents]
         text_embeddings = torch.from_numpy(text_embeddings).to(dtype)
         text_embeddings_numpy = text_embeddings.detach().numpy()
-        self.load_unet()
+        if(text_embeddings.shape[1] <= 64):
```

Comment:
We don't need to, since all unloading does is

```
del x
x = None
```

which works fine even if it's not loaded. I can change it for clarity's sake though.